diff --git a/scripts/react-native-xcode.sh b/scripts/react-native-xcode.sh
index 1dcdf55e190221c9b121df90452dd17ad47ff67b..482bb4f023e876f291c20fc2518113fcb6fc46d1 100755
--- a/scripts/react-native-xcode.sh
+++ b/scripts/react-native-xcode.sh
@@ -24,7 +24,7 @@ if [[ ! "$SKIP_BUNDLING_METRO_IP" && "$CONFIGURATION" = *Debug* && ! "$PLATFORM_
     IP=$(ifconfig | grep 'inet ' | grep -v ' 127.' | grep -v ' 169.254.' |cut -d\   -f2  | awk 'NR==1{print $1}')
   fi
 
-  echo "$IP" > "$DEST/ip.txt"
+  # echo "$IP" > "$DEST/ip.txt"
 fi
 
 if [[ "$SKIP_BUNDLING" ]]; then
